rust/main/chains/hyperlane-sealevel/src/log_meta_composer.rs

```diff
@@ -64,15 +63,17 @@ impl LogMetaComposer {
             )))?
         }
 
-        let (transaction_index, transaction_hash) =
-            transaction_hashes
-                .into_iter()
-                .next()
-                .ok_or(HyperlaneSealevelError::NoTransactions(format!(
+        let (transaction_index, transaction_hash, program_index) = transaction_hashes
+            .into_iter()
+            .next()
+            .ok_or(HyperlaneSealevelError::NoTransactions(format!(
                 "block which should contain {} transaction does not contain any after filtering",
                 self.transaction_description,
             )))?;
 
+        // Construct log index which will be increasing relative to block
```

Comment:
fwiw I view us as being able to define this however we want per VM, and don't think we necessarily need to replicate the EVM behavior here, but happy as it is